frontend/src/components/tables/PropertyDetailsTable.js

```diff
@@ -0,0 +1,247 @@
+import {StyledTableWithResults} from "../../styles/TableWithResults.style";
+import {StyledTableWithoutResults} from "../../styles/TableWithoutResults";
+
+export default function PropertyDetailsTable({property, units}) {
+
+    return (
+        <div>
+            <StyledTableWithResults>
+                <thead>
+                <tr>
+                    <th colSpan={3}>Purchase Price</th>
+                </tr>
+                </thead>
+                <tbody>
+                <tr>
+                    <td> Purchase Price</td>
+                    <td>{property.purchasePrice}</td>
+                    <td>{units.euro}</td>
+                </tr>
+                <tr>
+                    <td> Real estate agent fees</td>
+                    <td>{property.realEstateAgentFees}</td>
+                    <td>{units.euro}</td>
+                </tr>
+                <tr>
+                    <td> Real estate transfer tax</td>
+                    <td>{property.realEstateTransferTax}</td>
+                    <td>{units.euro}</td>
+                </tr>
+                <tr>
+                    <td> Notary fee</td>
+                    <td>{property.notaryFee}</td>
+                    <td>{units.euro}</td>
+                </tr>
+                </tbody>
+                <tfoot>
+                <tr>
+                    <th>Sum</th>
+                    <th>{property.overallPurchasePrice}</th>
+                    <th>{units.euro}</th>
+                </tr>
+                </tfoot>
+            </StyledTableWithResults>
```

Comment:
Extract to own component

Reply:
ok. i want to discuss this on 1:1 session.